src/components/Api.js

```diff
@@ -190,7 +190,7 @@
       });
   }
```

Comment:
```js

      .catch((err) => {
        console.error(err);
      })
```

It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very end of any request and this end is in `index.js`  after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.

Please, remove it from `Api` and add it in `index.js`.